projects/1.20/assets/vics-point-blank/pointblank/lang/zh_cn.json

```
@@ -0,0 +1,62 @@
+{
+  "itemGroup.pointblank.items": "搏枪斗械",
+  "key.categories.pointblank": "搏枪斗械",
+  "key.pointblack.reload": "装填枪械",
+  "key.pointblack.firemode": "切换开火模式",
+  "key.pointblack.inspect": "检视枪械",
+
+  "label.pointblank.ammo": "弹药",
+
+  "item.pointblank.glock17": "Glock-17 手枪",
+  "item.pointblank.glock18": "Glock-18 手枪",
+  "item.pointblank.m1911a1": "M1911A1 手枪",
+  "item.pointblank.m4a1": "M4A1 突击步枪",
+  "item.pointblank.m4a1mod1": "M4A1 MOD I 突击步枪",
+  "item.pointblank.star15": "ST AR-15 突击步枪",
+  "item.pointblank.m4sopmodii": "M4 SOPMOD II 突击步枪",
+  "item.pointblank.m16a1": "M16A1 突击步枪",
+  "item.pointblank.scarl": "Scar-L Tactical 突击步枪",
+  "item.pointblank.xm7": "XM 7突击步枪",
+  "item.pointblank.g36c": "G36C 突击步枪",

```

```suggestion
  "item.pointblank.g36c": "G36C突击步枪",
```
武器命名是有什么特殊规范吗

在1.16或更高版本下，英文字母、阿拉伯数字等非中文文本无需在前后添加空格。
具体可见我们的 [翻译规范](https://cfpa.site/TransRules/) 中 **04.排版** 一节下。
诸如这类的问题请在您翻译的所有文件中全部查明并修改。